common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/AbstractRootConcept.scala

```diff
@@ -22,101 +18,49 @@ object Concept {
   }
 }
 
-case class InstantRange(label: String,
-                        from: Instant,
+// We're not extending this yet, as we don't actually want it to be part of
+// the Display model as yet before we've started testing, but in future it
+// might extend AbstractConcept
+case class InstantRange(from: Instant,
                         to: Instant,
-                        inferred: Boolean)
-// We're not extending this yet, as we don't
-// actually want it to be part of the Display
-// model as yet before we've started testing.
-// extends AbstractConcept
+                        label: String = "",
+                        inferred: Boolean = false) {
+
+  def withInferred(inferred: Boolean): InstantRange =
+    InstantRange(from, to, label, inferred)
+
+  // TODO: is label necessary? appears to always be same as Period.label
```

Comment:
The reason for the `label` is it's required by the [`AbstractConcept`](https://github.com/wellcometrust/catalogue/blob/master/common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/AbstractRootConcept.scala#L12), which this will probably extend from.

It's what the rendering app requires so as not to have to work with the structured data for rendering (although I imagine they will on occasion).

ping @gestchild and @davidpmccormick to check in if that was their understanding?